app.json

```diff
@@ -115,6 +115,10 @@
       "description": "The log level for django",
       "required": false
     },
+    "EDX_ACCESS_TOKEN": {
```

Comment:
Per the discussion [here](https://github.com/mitodl/mitxpro/issues/1026), this should be `OPENEDX_GRADES_API_TOKEN`

Reply:
Updated